spec/AutomatedLinkAcceptance+services.yaml

```diff
@@ -1,87 +1,99 @@
 servers:
-## TODO: Search and replace all "xx" in the uuid by the official abbreviation of the ApplicationName
+
```

Comment:
all "_- service-name:_ ..." entries have been changed to "_- operation-name:_ ..." in latest version of ApplicationPattern, so that change also needs to be done here.

Reply:
all entries "-service-name:" were replaced by "-operation-name:"